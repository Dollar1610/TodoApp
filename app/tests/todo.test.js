import React from 'react';
import { shallow } from 'enzyme';
import Todo from './../todo';


describe('<Todo />', () => {
  it('snapshot for Todo component', () => {
    const snapshotTodo = shallow(<Todo />).html();
    expect(snapshotTodo).toMatchSnapshot();
  });
});
