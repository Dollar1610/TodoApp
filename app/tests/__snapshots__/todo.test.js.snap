// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Todo /> snapshot for Todo component 1`] = `"<section class=\\"section\\"><div class=\\"container\\"><header><div class=\\"product-name\\">Product name</div><div class=\\"mail\\">test@mail.ru</div></header><nav class=\\"level is-mobile\\"><div class=\\"level-item has-text-centered\\"><div class=\\"application field\\"><div class=\\"field\\"><input type=\\"text\\" id=\\"inputText\\" class=\\"application_input input\\" placeholder=\\"Enter text\\"/></div><div class=\\"list-items\\">Список пуст</div><button class=\\"application_button button\\">Add</button><button class=\\"application_button button\\">Remove</button></div></div></nav><footer><div class=\\"copyright\\">copyright</div></footer></div></section>"`;
